core/src/main/kotlin/com/exactpro/th2/read/db/core/impl/DataBaseMonitorServiceImpl.kt

```diff
@@ -108,44 +113,53 @@ class DataBaseMonitorServiceImpl(
     private suspend fun poolUpdates(
         dataSourceId: DataSourceId,
         startFromLastReadRow: Boolean,
+        resetStateParameters: ResetState,
         initQueryId: QueryId?,
         initParameters: QueryParametersValues,
         useColumns: Set<String>,
         updateParameters: QueryParametersValues,
-        interval: Duration,
+        interval: Long,
```

Comment:
Why get rid of the Duration here? The Duration provides more information than some Long value. Is it milliseconds  or nanosecods?

Reply:
This private method which converts `interval: Duration` to millisecond only.
We can pass the duration type in future if it need

Reply:
This is exactly why the `Duration` is passed to this method instead of Long - **this** method converts duration into the unit **it needs**. Now another method does that and passes an `interval` parameter with the Long type. What does this parameter contain? Seconds? Milliseconds? Nanosecods? I cannot answer this question without looking into `delay` method doc. If you are going to use Long we can at least rename the parameter to indicate what is expected by the method

Reply:
OK, I will correct name in this method to `intervalMilliseconds`